README.md

```diff
@@ -56,6 +56,9 @@ Synopsis
 
         --resolve-ipv6      Make the nginx resolver lookup both IPv4 and IPv6 addresses
 
+        --shdict NAME       Create the specified lua shared dicts in the http
```

Comment:
Sorry, I've had a closer look and found that the real syntax is actually `--shdict "NAME SIZE"` here?